helper.rb

```diff
@@ -0,0 +1,14 @@
+require "talk_like_a_pirate"
+
+
+def piratize(normal_sentence)
```

Comment:
The parameter is misleading. Technically a user can pass in anything including hashes, arrays, symbols and custom objects as the challenge suggests. I would rename to something more generic.